test/prototype_transforms_kernel_infos.py

```diff
@@ -1171,7 +1178,8 @@ def reference_inputs_pad_bounding_box():
 
 
 def sample_inputs_perspective_image_tensor():
-    for image_loader in make_image_loaders(sizes=["random"]):
+    # Skip tests on dtype float64, otherwise scripted vs eager are failing
```

Comment:
Failing how? In case of tolerance issues, use the fix I proposed above.

Reply:
Indeed it was tolerance issue but larger than atol/rtol you proposed above. For example
```
_______________________________________________________ TestKernels.test_scripted_vs_eager[cuda-perspective_image_tensor-251] ________________________________________________________
Traceback (most recent call last):                                                                                                                                                    
  File "/vision/test/test_prototype_transforms_functional.py", line 141, in test_scripted_vs_eager                                                                                    
    assert_close(                                                                                                                                                                     
  File "/vision/test/prototype_common_utils.py", line 124, in assert_close                                                                                                            
    raise error_metas[0].to_error(msg)                                                                                                                                                
AssertionError: Tensor-likes are not close!                                                                                                                                           
                                                                                                                                                                                      
Mismatched elements: 6785 / 19968 (34.0%)                                                                                                                                             
Greatest absolute difference: 2.171263325467798e-06 at index (1, 0, 0, 30, 8) (up to 1e-07 allowed)                                                                                   
Greatest relative difference: 0.00013440363192679722 at index (0, 1, 0, 21, 15) (up to 1e-07 allowed)                                                                                 
Failure happened for the following parameters:                                                                                                                                        
                                                                                                                                                                                      
tensor(shape=[2, 3, 4, 32, 26], dtype=torch.float64, device=cuda:0)                                                                                                                   
tensor(shape=[2, 3, 4, 32, 26], dtype=torch.float64, device=cuda:0)                                                                                                                   
None                                                                                                                                                                                  
None                                                                                                                                                                                  
fill=[0.1, 0.2, 0.30000000000000004, 0.4]                                                                                                                                             
coefficients=[1.2405, 0.1772, -6.9113, 0.0463, 1.251, -5.235, 0.00013, 0.0018] 
```
or 
```
AssertionError: Tensor-likes are not close!                                                                                                                                           
                                                                                                                                                                                      
Mismatched elements: 4001 / 10296 (38.9%)                                                                                                                                             
Greatest absolute difference: 1.62549076154761e-06 at index (1, 1, 0, 25, 2) (up to 1e-07 allowed)                                                                                    
Greatest relative difference: 1.8553701461158903e-05 at index (1, 2, 0, 8, 10) (up to 1e-07 allowed)                                                                                  
Failure happened for the following parameters:                                                                                                                                        
                                                                                                                                                                                      
tensor(shape=[2, 3, 3, 26, 22], dtype=torch.float64, device=cpu)                                                                                                                      
tensor(shape=[2, 3, 3, 26, 22], dtype=torch.float64, device=cpu)                                                                                                                      
None                                                                                                                                                                                  
None                                                                                                                                                                                  
fill=[0.1, 0.2, 0.30000000000000004]                                                                                                                                                  
coefficients=[1.2405, 0.1772, -6.9113, 0.0463, 1.251, -5.235, 0.00013, 0.0018] 
```

There are 96 failed tests